frontend/views/utils/captureLogs.js

```diff
@@ -0,0 +1,102 @@
+import sbp from '~/shared/sbp.js'
+import { CAPTURED_LOGS } from '~/frontend/utils/events.js'
+
+const SIZE_LIMIT_KB = 100
+const isNewSession = !sessionStorage.getItem('NEW_SESSION')
+const initialLogs = localStorage.getItem(CAPTURED_LOGS)
+
+// Mark in localStorage that is a new visit (and session)
+if (isNewSession) { sessionStorage.setItem('NEW_SESSION', 1) }
```

Comment:
What is this `sessionStorage` stuff? I don't think we need it.

Reply:
It helps us understand if the user is visiting the page as a brand new session (ex: a new tab) or if it's just a new visit (a page refresh). It might be useful.

- [Diff between localstorage and sessionstorage](https://stackoverflow.com/questions/19867599/what-is-the-difference-between-localstorage-sessionstorage-session-and-cookies)